config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -0,0 +1,62 @@
+# This configuration file is for the plate and tube purposes required for the
+# high throughput scRNA Core Cell Extraction pipeline
+---
+###
+# scRNA Core Cell Extraction Arraying
+###
+# Donor Vacutainer tubes
+LRC Blood Vac:
+  :asset_type: tube
+  :stock_plate: false
+  :input_plate: false
+  :target: SampleTube
+  :type: Tube::Purpose
+  :presenter_class: Presenters::VacTubePresenter
+  :creator_class: LabwareCreators::Uncreatable
+  :default_printer_type: :tube
+###
+# scRNA Core Cell Extraction Blood Pipeline
+###
+# Input plate for Hamilton PBMC isolation protocol
+# LRC Blood Vac tubes are also arrayed into LRC Blood Bank plates (DPL-073)
+LRC Blood Bank:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :creator_class:
+    name: LabwareCreators::MultiStampTubes
+    params:
+      allow_tube_duplicates: yes
+  :submission_options:
+    scRNA core cell extraction:
+      template_name: 'Limber - scRNA Core Cell Extraction'
+      request_options:
+        fragment_size_required_from: '200'
+        fragment_size_required_to: '800'
+      allowed_extra_barcodes: false
+  :file_links:
+    - name: 'Download Scanned Tube Layout for LRC Blood Bank Plate'
+      id: scrna_core_cell_extraction_sample_arraying_tube_layout
+# LRC Blood Bank plates are stamped into LRC PBMC Bank plates
+# Output plate from Hamilton PBMC isolation protocol / input plate for the cell counting and cell banking protocols
+LRC PBMC Bank:
+  :asset_type: plate
+  :file_links:
+    - name: 'Download Hamilton LRC PBMC Bank plate to LRC Bank Seq tubes CSV'
```

Comment:
Were you going to change the name of this file (to go to both tube types, not just one)? Or leave it for DPL-878?